frontend/src/utils/account-send.ts

```diff
@@ -52,6 +52,16 @@ export type StoreSendArgs = {
   accountDataToEncrypt: AccountDataToEncrypt;
 };
 
+type BatchStoreSendArgs = {
+  chainId: number;
+  viewingPrivateKey: string;
+  advancedMode: boolean;
+  usePublicKeyChecked: boolean;
+  batches: { batch: SendBatch; pubKey: string }[];
+  batchTxHash: string;
+  senderAddress: string;
+};
```

Comment:
Can this just be `type BatchStoreSendArgs = StoreSendArgs[]`? And really in that case we don't even need a new `BatchStoreSendArgs` type. Arguably it's less efficient since we're duplicating tx hash/etc, but that difference will be negligible, and it feels simpler that way

Reply:
In this case I think we could combine both functions since we are already iterating through all of the sends before hand to lookup the public key. The only downside is duplication of data along with transforming the batch send data which we are already doing somewhat.

Reply:
Instead I removed batch store and consolidated here https://github.com/ScopeLift/umbra-protocol/pull/524/commits/6e68909bd31bc23e5bcb2037a89dd643a8faeb5b

Reply:
nice this looks a lot simpler now, thanks!